docs/entities/renderer.md

```diff
@@ -159,7 +160,7 @@ Let's start by creating an entity model class:
 public class MyEntityModel extends EntityModel<MyEntityRenderState> {}
 ```
 
-Note that in the above example, we directly extend `EntityModel`; depending on your use case, it might be more appropriate to use one of the subclasses instead. When creating a new model, it is recommended you have a look at whatever existing model is closest to your use case, and then work from there.
+Note that in the above example, we directly extend `EntityModel`; depending on your use case, it might be more appropriate to use one of the subclasses or even `Model` instead. When creating a new model, it is recommended you have a look at whatever existing model is closest to your use case, and then work from there.
```

Comment:
```suggestion
Note that in the above example, we directly extend `EntityModel`; depending on your use case, it might be more appropriate to use one of the subclasses, or even just `Model` or a non-entity-related subclass of `Model`, instead. When creating a new model, it is recommended you have a look at whatever existing model is closest to your use case, and then work from there.
```